app/javascript/packs/colorPreview.js

```diff
@@ -1,52 +1,47 @@
+/* eslint-disable no-param-reassign */
 
-function initPreview() {
-  const colorField = {
-    // input type="color"
-    bgColor: document.getElementById('bg-color-colorfield'),
-    textColor: document.getElementById('text-color-colorfield'),
-  };
-  
-  const textField = {
-    // input type="text"
-    bgColor: document.getElementById('bg-color-textfield'),
-    textColor: document.getElementById('text-color-textfield'),
-  };
-  
-  const preview = document.getElementById('color-select-preview-logo');
-  
-  // Assigns input[type='text'] values to input[type='color']
-  function updateColorFields() {
-    colorField.bgColor.value = textField.bgColor.value;
-    colorField.textColor.value = textField.textColor.value;
-  }
-  
-  // Updating text fields when color fields are changed
-  function updateTextFields() {
-    textField.bgColor.value = colorField.bgColor.value;
-    textField.textColor.value = colorField.textColor.value;
-  }
-  
-  // Updates Preview Colors
-  function updatePreview() {
-    preview.style.backgroundColor = textField.bgColor.value;
-    preview.style.fill = textField.textColor.value;
-  }
-  
-  // Event Watchers
-  // When color fields change -> updateTextField values and refresh preview
-  function watchColorFields() {
-    updateTextFields();
+const preview = document.getElementById('color-select-preview-logo');
```

Comment:
not in love with this global object but i understand why use outer scope for the event handlers. maybe you could just get the element where needed instead of the outer scope?

Reply:
Yeah, sounds good.